embedx/config.schema.json

```
@@ -460,7 +460,8 @@
             "linkedin",
             "linkedin_v2",
             "lark",
-            "x"
+            "x",
+            "line_v2_1"
```

See my other comment

```suggestion
            "line"
```